src/Service/HttpService.ts

```diff
@@ -120,7 +120,11 @@ export class HttpService implements HttpServiceInterface {
 
     this.applyResponseHeaders(serviceResponse, response)
 
-    response.status(serviceResponse.status).send(serviceResponse.data)
+    if (serviceResponse.request._redirectable._redirectCount > 0) {
```

Comment:
Hmm not sure but I think `axios` will follow redirects by default. Based on this:

```
  // `maxRedirects` defines the maximum number of redirects to follow in node.js.
  // If set to 0, no redirects will be followed.
  maxRedirects: 5, // default
  ```
  
Resource: https://axios-http.com/docs/req_config

Reply:
Tested this but the thing is, if axios follow the redirect, the api-gateway endpoint returns a 200 instead of a 301 to the browser. This means the browser tries to load the HTML returned by api-gateway, and when it loads any asset with a relative path in the HTML, it uses api-gateway's URL (which won't work since we need it to load them from website). So basically what we want here is for api-gateway not to follow the redirect, but return it as a 301